app/presenters/report_presenter.rb

```diff
@@ -29,8 +29,8 @@ def priority_percentage(priority:)
   end
 
   def due_defects_by_priority(priority:)
-    defects = defects_by_priority(priority: priority)
-    defects.where('target_completion_date >= ?', Date.current)
+    open_defects = defects_by_priority(priority: priority).open
+    open_defects.where('target_completion_date >= ?', Date.current)
```

Comment:
This is kind of hard to read. Could we do something like:

```suggestion
    defects = defects_by_priority(priority: priority)
    defects.open.where('target_completion_date >= ?', Date.current)
```